dev/breeze/tests/test_packages.py

```diff
@@ -221,6 +221,7 @@ def test_get_documentation_package_path():
             """
     "apache-airflow-providers-common-sql>=1.17.0",
     "apache-airflow>=2.8.0",
+    "asyncpg>=0.30.0",
```

Comment:
psycopg 3 also has async support but might conflict with psycopg 2 already installed.

https://www.psycopg.org/psycopg3/docs/advanced/async.html

https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg

Reply:
> psycopg 3 also has async support but might conflict with psycopg 2 already installed.
> 
> https://www.psycopg.org/psycopg3/docs/advanced/async.html
> 
> https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg

True, yeah.  We could in future look at allowing diff libs.

Reply:
Benchmark, so take with a pinch of salt, but https://pypi.org/project/asyncpg/

> In our testing asyncpg is, on average, 5x faster than psycopg3.
>
> ![](https://pypi-camo.freetls.fastly.net/cfd9ddf54ddaa3b325b4df42aec46c44e32219c2/68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f4d61676963537461636b2f6173796e6370672f6d61737465722f706572666f726d616e63652e706e673f66646463613430616230)

That's def a big enough of a change that we should see if we notice any difference